crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -80,7 +82,16 @@ pub fn all_tuples(input: TokenStream) -> TokenStream {
     })
 }
 
-#[proc_macro_derive(Bundle)]
+#[derive(Eq, PartialEq)]
```

Comment:
Any specific reason `Eq/PartialEq` are derived?
We never use equality with this type.

Reply:
true! that part of the code is gone now...